tests/cephfs/cephfs_bugs/validate_ceph_ops_wtih_fs_fail.py

```diff
@@ -0,0 +1,59 @@
+import traceback
+
+from ceph.ceph import CommandFailed
+from ceph.parallel import parallel
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    try:
+        """
+        CEPH-83589266 - Validate Ceph commands when ceph file system is in failed state
+
+
+        """
+        tc = "CEPH-83583721"
+        log.info("Running cephfs %s test case" % (tc))
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        clients = ceph_cluster.get_ceph_objects("client")
+        build = config.get("build", config.get("rhbuild"))
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        client1 = clients[0]
+        fs_name = "cephfs_fail"
+        fs_util.create_fs(client1, fs_name)
+        fs_util.wait_for_mds_process(clients[0], fs_name)
+        client1.exec_command(sudo=True, cmd=f"ceph fs fail {fs_name}")
+
+        with parallel() as p:
+            p.spawn(
+                client1.exec_command,
+                sudo=True,
+                cmd=f"ceph fs subvolume create {fs_name} subvol_1;",
+                check_ec=False,
+                timeout=720,
+            )
+            p.spawn(client1.exec_command, sudo=True, cmd="ceph fs status")
+            p.spawn(client1.exec_command, sudo=True, cmd="ceph mgr stat")
+            p.spawn(client1.exec_command, sudo=True, cmd="ceph pg stat")
+            for result in p:
+                print(result)
```

Comment:
Should we also validate result, as in, if result is failure in case of other ceph stat cmds, then we should return 1?
To know if ceph cmds suceeds with ceph fs fail and volume create stuck?

Reply:
Hi Suma,
if any of the above command fails it will fail with CommandFailed and hence test case will fail

Regards,
Amarnath